README.md

```diff
@@ -1,2 +1,90 @@
-# GitHubAiAPI
-This is Example to test AI plugin for automatic code review
+# GitHub Pull Request AI Reviewer
+
+## Overview
+
+The GitHub Pull Request AI Reviewer is a tool designed to facilitate code reviews by leveraging AI to provide preliminary insights on pull requests. This project includes a Go-based backend server and an HTML frontend interface to interact with GitHub repositories, manage pull requests, and generate review comments.
+
+## Features
+
+- **Repository Existence Check:** Verifies if a specified repository exists.
+- **Pull Request Management:** Fetches the latest pull request and its associated diff files.
+- **AI Review Integration:** Sends pull request diffs to an AI system for review and displays the results.
+- **Error Handling:** Gracefully handles scenarios like missing pull requests or diff files.
+
+## Owner
+Saurabh Chauhan
+
+## Getting Started
+
+### Prerequisites
+
+- Go (version 1.16 or higher)
+- Git
+- A GitHub account with an API token
+- Access to the AI review system (URL: `http://ah-knathan-m.dhcp.mathworks.com:8082/mwai/api`)
+
+### Installation
+
+1. **Clone the repository:**
```

Comment:
The GITHUB_TOKEN should not be exposed in the README. Consider replacing it with a placeholder.